unitTests/ui/FieldTableTest.ts

```diff
@@ -64,6 +64,10 @@ export function FieldTableTest() {
           return $$(test.env.root).find('.coveo-field-table-toggle-container');
         }
 
+        function toggleCaption() {
+          return $$(test.env.root).find('.coveo-field-table-toggle-caption');
+        }
```

Comment:
@btaillon, I will wait for you to merge and then adjust my function to use yours.

Reply:
Alright, I'll mention you when I merged.